src/autopas/containers/verletClusterLists/ClusterTower.h

```diff
@@ -187,22 +187,37 @@ class ClusterTower : public ParticleCell<Particle> {
   [[nodiscard]] size_t getNumTailDummyParticles() const { return _numDummyParticles; }
 
   /**
-   * @copydoc getNumActualParticles()
+   * Get the number of all particles stored in this tower (owned, halo and dummy).
+   * @return number of particles stored in this tower (owned, halo and dummy).
    */
-  [[nodiscard]] unsigned long numParticles() const override { return getNumActualParticles(); }
+  [[nodiscard]] unsigned long size() const override { return getNumActualParticles(); }
 
   /**
-   * Returns the number of particles in the tower before the filler dummies.
-   * There might still be particles that were marked as dummies due to deletion.
-   * @return
+   * @copydoc autopas::ParticleCell::getNumberOfParticles()
    */
-  [[nodiscard]] size_t getNumActualParticles() const { return getNumAllParticles() - getNumTailDummyParticles(); }
+  [[nodiscard]] unsigned long getNumberOfParticles(IteratorBehavior behavior) const override {
+    return _particlesStorage.getNumberOfParticles(behavior);
+  }
 
   /**
-   * Returns the size of the internal particle storage aka. the total number of particles incl. dummies.
-   * @return
+   * Get the number of all particles saved in the tower without dummies for data structure reasons (owned + halo +
+   * dummies for non-datastructure purposes).
+   * @return Number of all particles saved in the tower without dummies for data structure reasons (owned + halo +
+   * dummies for non-datastructure purposes).
```

Comment:
Am I correct that the "dummies for data structure reasons" are the tailing dummies to ensure that every cluster is filled? If so, clarify that the number of particles excludes tailing dummies.

Reply:
Yes, these are the tailing dummies. 
Done in [37e2621](https://github.com/AutoPas/AutoPas/pull/782/commits/37e26218410380f7bc586d66bd83b316d23a3d28)